stdlib/testing/testdata/holt_winters.flux

```diff
@@ -0,0 +1,1292 @@
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T01:18:00.000000000Z,5.341
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T01:24:00.000000000Z,5.325
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T01:30:00.000000000Z,5.387
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T01:36:00.000000000Z,5.299
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T01:42:00.000000000Z,5.302
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T01:48:00.000000000Z,5.236
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T01:54:00.000000000Z,5.164
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:00:00.000000000Z,5.171
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:06:00.000000000Z,5.098
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:12:00.000000000Z,5.059
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:18:00.000000000Z,4.964
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:24:00.000000000Z,4.911
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:30:00.000000000Z,4.872
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:36:00.000000000Z,4.806
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:42:00.000000000Z,4.777
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:48:00.000000000Z,4.662
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T02:54:00.000000000Z,4.551
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T03:00:00.000000000Z,4.495
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T03:06:00.000000000Z,4.432
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T03:12:00.000000000Z,4.367
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T03:18:00.000000000Z,4.275
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T03:24:00.000000000Z,4.265
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T03:30:00.000000000Z,4.111
+,,0,2015-08-22T22:12:00.000000000Z,2015-08-28T03:01:00.000000000Z,water_level,water,2015-08-25T03:36:00.000000000Z,3.986
+,,0,2015-08-22T2
```

Comment:
I'd really like to find out why this is the case.  I haven't yet, but will keep looking,